Docs/ConfigOptions.md

```diff
@@ -308,6 +309,65 @@ commentOptions:
 
 <br/>
 
+## packages
+
+**Type:** [PackageDeclaration]<br />
+**Default:** `[]`
+
+Additional remote Swift Package imports.
```

Comment:
I was going to add more documentation to say that you can use this alongside `entities/paths.imports` but that can wait till https://github.com/CreateAPI/CreateAPI/issues/114 so that it's even more descriptive.